vendor/src/github.com/docker/distribution/registry/client/auth/session.go

```diff
@@ -110,7 +110,7 @@ func (ts tokenScope) String() string {
 
 // NewTokenHandler creates a new AuthenicationHandler which supports
 // fetching tokens from a remote token server.
-func NewTokenHandler(transport http.RoundTripper, creds CredentialStore, scope string, actions ...string) AuthenticationHandler {
+func NewTokenHandler(transport http.RoundTripper, creds CredentialStore, scope string, actions []string) AuthenticationHandler {
```

Comment:
Unnecessary change that may have a wider impact. Would prefer this is taken out.

(Also, this is not the appropriate place to make this change, anything in /vendor needs to be changed at the source).